docs/source/user_guide/runtime-image-conf.md

```diff
@@ -23,7 +23,8 @@ A runtime image configuration identifies a container image that Elyra can utiliz
 
 A runtime image configuration is associated with a container image that must meet these prerequisites:
 
-- The image must be stored in a public container registry (no authentication required) (e.g. [https://hub.docker.com](https://hub.docker.com)).
+- The image is stored in a container registry in a public or private network that the container platform in which the pipeline is executed can connect to. Examples of such registries are hub.docker.com or a self-managed registry in an intranet environment.
```

Comment:
Same here...
```suggestion
- The image is stored in a container registry in a public or private network that the container platform in which the pipeline is executed can connect to. Examples of such registries are [https://hub.docker.com](https://hub.docker.com) or a self-managed registry in an intranet environment.
```

Reply:
Ugh.  I'm sorry.  I didn't intend for the `https://` prefix to sneak into the link's text.  I think just having `hub.docker.com` is best (sans scheme).